common/app/views/fragments/email/signup/emailSignUp.scala.html

```diff
@@ -42,6 +42,7 @@
 
 @form = {
     <form action="@LinkTo("/email")" method="post" id="@formId" class="@formClass" data-email-form-type="@componentClass", data-email-list-name="@listName">
+        @helper.CSRF.formField
```

Comment:
Huh so `csrfAddToken` does nothing for this use case?

Reply:
Yeah, I Think csrfAddToken adds the cookie value SC_GU_CSRF (or whatever it's called) which is sent checked: the form field will be included in the body so this will also pass the check (just testing it on code though to be sure).